docs/site/Calling-other-APIs-and-Web-Services.md

```diff
@@ -60,19 +60,6 @@ Install the REST connector used by the new datasource:
 $ npm install --save loopback-connector-rest
 ```
 
-### Bind data sources to the context
-
-```ts
-import {Context} from '@loopback/context';
-
-const context = new Context();
-context.bind('dataSources.geoService').to(ds);
-```
-
-**NOTE**: Once we start to support declarative datasources with
-`@loopback/boot`, the datasource configuration files can be dropped into
-`src/datasources` to be discovered and bound automatically.
```

Comment:
This doc fix is not strictly related to the changes made by my pull request, the section should have been removed when the datasource booter was added. Since it's just a small change, I didn't feel like creating a new commit or a new pull request.